### PR TITLE
Feature to use daycare finance handler always

### PR DESCRIPTION
**Pull request by reynders**

### CODE REVIEW

File: service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/VoucherValueDecisionIntegrationTest.kt [hunk 1]
```diff
@@ -448,6 +449,40 @@ class VoucherValueDecisionIntegrationTest : FullApplicationTest(resetDbBeforeEac
...
     }
 
+    @Test
+    fun `VoucherValueDecision handler is set to the user when decision is not normal`() {
+        val approvedDecision = createReliefDecision(false)
+        assertEquals(testDecisionMaker_1.id.raw, approvedDecision.decisionHandler)
+    }
+
+    @Test
+    fun `VoucherValueDecision handler is set to the daycare handler when forced when decision is not normal`() {
```
Comment (hruoho) [inline service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/VoucherValueDecisionIntegrationTest.kt:459]: Is this named as you intended?

File: service/src/main/kotlin/fi/espoo/evaka/shared/FeatureConfig.kt [hunk 2]
```diff
@@ -41,4 +41,12 @@ data class FeatureConfig(
      * and all the operating days are marked either sick leaves or planned absences.
      */
     val freeSickLeaveOnContractDays: Boolean,
+
+    /** Whether to always use the marked daycare finance decision handler
+     *
+     *  If true, regardless of situation use the handler marked in the daycare settings
+     *  as the handler of a decision.
+     *  If false, use the current user as the handler as speficied in the code
```
Comment (hruoho) [inline service/src/main/kotlin/fi/espoo/evaka/shared/FeatureConfig.kt:49]:
```suggestion
     *  If true, use the handler marked in the daycare settings, whenever available,
     *  as the handler of a decision.
     *  If false, use either the current user or the handler marked 
     *  in the daycare settings, depending on the case.
```

**Review by hruoho — APPROVED** [review]
Looks good to me!

File: service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/VoucherValueDecisionIntegrationTest.kt (hunk 1 above)
Comment (reynders) [reply service/src/integrationTest/kotlin/fi/espoo/evaka/invoicing/VoucherValueDecisionIntegrationTest.kt:459]: Unfortunately yes :D.

**Pull request merged**